December report (#35)
uk-tre/website: docs/events/wg_workshops/2023-12-05-december-meeting/workshop-satre.md

- https://www.youtube.com/watch?v=auExNHEGwcc
- https://www.youtube.com/watch?v=kzUU5ljII0Q

### Target audience

Users, researchers, implementers of TREs and of course any public members who wish to help support the transparency and openness of the project.

## Session

### Summary

SATRE was introduced to the room, before the idea of a SATRE working group was discussed, and how it could align with ongoing work in the UK TRE community.

Alignment between UKSA and SDE accreditation was also discussed, as well as organisations carrying out SATRE evaluation.

Next steps focused on a January meeting for a working group next steps, with a focus on helping other orgs evaluate themselves

### Raw notes

#### What is SATRE?

- DARE UK funded project
- Focus to be a community project

> - Led by HIC, Dundee; Alan Turing; UCL

Reviewer: ```suggestion
- Led by HIC, Dundee; The Alan Turing Institute; UCL
```

- https://satre-specification.readthedocs.io/en/latest/

> - Leanr and explore

Reviewer: ```suggestion
- Learn and explore
```

What capacity is required to create a viable SATRE Working Group? What tasks do we need to complete?

- The NWSDE's Technical Design Authority (TDA) has a team looking at compliance of our tech and processes with SATRE (while we wait to find out more about the UKSA accreditation framework)
- What happens next?
  - Repo could stagnate
  - Funding could support ongoing maintenance/development
    - What is needed to smooth out support
  - What voices were missed out in the first phase?
- Similar questions have been asked about UK TRE Community
- Working groups fundamental, strongly hope to have support mechanisms available in the next phase
  - SATRE Fundamental to day job of defining a federated architecture of TRE
  - Pulled together driver projects to form architecture, SATRE forms part of that ongoing picture
  - Third avenue is feeding into emerging working group at Research Data Alliance. See [Recent Plenary BoF session](https://www.rd-alliance.org/trusted-research-environments-sensitive-data-fairness-closed-data-and-processes) and [draft WG charter](https://docs.google.com/document/d/1877OtQyZ46QCHVZ8_1QqRgZJPXyImZdW1qYVyKItMSQ/edit).

> - Great that SATRE invovled public, reflect this in the specification would be good for funding, and to highlight directly in the specifications where the project has reflected public/patient requests.

Reviewer: ```suggestion
- Great that SATRE involved public, reflect this in the specification would be good for funding, and to highlight directly in the specifications where the project has reflected public/patient requests.
```

> - Great that SATRE invovled public, reflect this in the specification would be good for funding, and to highlight directly in the specifications where the project has reflected public/patient requests.

Reviewer 2: ```suggestion
- Great that SATRE involved public, reflecting this in the specification would be good for funding, and to highlight directly in the specifications where the project has reflected public/patient requests.
```

- There are specific statements that came out of public involvement.
- re above points, aligned with TRE Community application but understanding needs for funding and gaps is key.

> - Build on public engagement, on that angle, nature of spec neeeds be broad and high level, important in the public engagment section. Overlaps with other work in the wider community.

Reviewer 2: ```suggestion
- Build on public engagement, on that angle, nature of spec neeeds be broad and high level, important in the public engagement section. Overlaps with other work in the wider community.
```

- People present happy to Chair a working group
- Happy to co-Chair while aware of appearance of conflict.

> - Another call for the SATRE working group before Christmas to be ready for January and identify key prioirities

Reviewer 2: ```suggestion
- Another call for the SATRE working group before Christmas to be ready for January and identify key priorities
```

> - Happy to share what they are doing with the SATRE team to get feedback on theri approach

Reviewer 2: ```suggestion
- Happy to share what they are doing with the SATRE team to get feedback on their approach
```

> How do SDE Accreditation framework from UKSA and SATRE fit together?

Reviewer 2: ```suggestion
How do SDE Accreditation frameworks from UKSA and SATRE fit together?
```

- And how do the UKSA/NHS conversations fit?
  - Anyone know how those are going?

Thoughts/comments on SATRE evaluation?

> - NWSDE are doing SATREfication of their TRE using github rather than spreadsheet

Reviewer 2: ```suggestion
- NWSDE are doing SATREfication of their TRE using GitHub rather than spreadsheet
```

  - Have an issue for each statement
  - Aim to get to green on mandatory/amber with exception in place in first phase
  - Have created [issue to define spec in a machine readable format](https://github.com/sa-tre/satre-specification/issues/254)

> - The SATRE pointing system could have subsection SATRE-PPI scoring how many of the public requirements are being met

Reviewer 2: ```suggestion
- The SATRE scoring system could have subsection SATRE-PPI scoring how many of the public requirements are being met
```

- Consensus there is a lot of value in having evaluation shared and exploring how others have scored but also process that goes into evaluation.

### Next steps

- Book a time in January for next meeting
- Focus on evaluations with other groups
- PIE - overlaps with PEDRI and continue engagements
- Get alignment with Research Data Alliance?

> - CC nomiated as chair to lead SATRE working group and explore means of succession funding

Reviewer 2: ```suggestion
- CC nominated as chair to lead SATRE working group and explore means of succession funding
```
